### PR TITLE
baconian cryptography in javascript

### PR DESCRIPTION
**Pull request by tbhaxor**
**Fixes issue:** None

**Changes:** Added baconian program in javascript

### CODE REVIEW

File: code/cryptography/src/baconian_cipher/baconian.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+function baconianEncrypt(plainText) {
+    baconianCodes = {
```
Comment (arnavb) [inline code/cryptography/src/baconian_cipher/baconian.js]: Use let.

File: code/cryptography/src/baconian_cipher/baconian.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        'a': 'AAAAA', 'b': 'AAAAB', 'c': 'AAABA', 'd':
+            'AAABB', 'e': 'AABAA', 'f': 'AABAB', 'g': 'AABBA', 'h': 'AABBB',
+        'i': 'ABAAA', 'j': 'BBBAA', 'k': 'ABAAB', 'l': 'ABABA', 'm':
+            'ABABB', 'n': 'ABBAA', 'o': 'ABBAB', 'p': 'ABBBA', 'q': 'ABBBB',
+        'r': 'BAAAA', 's': 'BAAAB', 't': 'BAABA', 'u': 'BAABB', 'v':
+            'BBBAB', 'w': 'BABAA', 'x': 'BABAB', 'y': 'BABBA', 'z': 'BABBB'
+    };
+
+    let cipherText = "";
+    [...plainText].forEach(char => {
```
Comment (arnavb) [inline code/cryptography/src/baconian_cipher/baconian.js]: I would just omit the braces; `forEach` doesn't use the return value anyways.

Comment (tbhaxor) [reply code/cryptography/src/baconian_cipher/baconian.js]: will u plz make it clear what do you want ?
because code is working fine at my system

Comment (danos7777) [reply code/cryptography/src/baconian_cipher/baconian.js]: I have never done this just trying to learn how.after my last three phones got hacked and want to learn how I can stop it.and add more securury

**Comment by tbhaxor** [comment]
@arnavb done :+1:

**Comment by tbhaxor** [comment]
@AdiChat plz review it too :smiley:

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**